json_serializable/test/json_serializable_test.dart

```diff
@@ -13,6 +13,16 @@ import 'package:test/test.dart';
 import 'analysis_utils.dart';
 import 'test_file_utils.dart';
 
+Matcher _matcherFromShouldGenerateAnnotation(ConstantReader reader) {
+  var expectedOutput = reader.read('expectedOutput').stringValue;
+  var isContains = reader.read('contains').boolValue;
+
+  if (isContains) {
```

Comment:
[optional] consider using ternary conditional since the interesting bit is _what_ it is, not the side effects.

Reply:
Will do in follow-up